### ISSUE TITLE
missing-value codes not inferred for a CTD file

### ISSUE THREAD

**Issue by dankelley** [dankelley/oce#1195]
```R
> d<-read.oce("18HU20130507_00235_00001_ct1.csv")
> summary(d)
CTD Summary
-----------

* File:                /Users/kelley/git/oce/tests/testthat/local_data/18HU20130507_00235_00001_ct1.csv
* Institute:           SIO
* Station:             235
* Location:            42.833N  61.737W 
* Water depth:         1070
* Data

                      Min. Mean   Max.   Dim. OriginalName
    pressure [dbar]   5    553    1101   1097 CTDPRS      
    temperature       -999 5.2551 11.6   1097 CTDTMP      
    salinity [PSS-78] -999 33.943 35.267 1097 CTDSAL      
    oxygen [μmol/kg]  -999 223.7  297.7  1097 CTDOXY      

* Data-quality Flags

    pressure:    "2" 1097
    temperature: "2" 1096, "9" 1
    salinity:    "2" 1096, "9" 1
    oxygen:      "2" 1093, "9" 4

* Processing Log
    - 2017-02-17 19:19:57 UTC: `create 'ctd' object`
    - 2017-02-17 19:19:57 UTC: `read.oce(file = "18HU20130507_00235_00001_ct1.csv")`

```

**Comment by dankelley** [comment]
Luckily, the flags are being handles right.
```R
> dd<-handleFlags(d)
> summary(dd)
CTD Summary
-----------

* File:                /Users/kelley/git/oce/tests/testthat/local_data/18HU20130507_00235_00001_ct1.csv
* Institute:           SIO
* Station:             235
* Location:            42.833N  61.737W 
* Water depth:         1070
* Data

                      Min.   Mean   Max.   Dim. OriginalName
    pressure [dbar]   5      553    1101   1097 CTDPRS      
    temperature       4.2675 6.1713 11.6   1097 CTDTMP      
    salinity [PSS-78] 33.128 34.885 35.267 1097 CTDSAL      
    oxygen [μmol/kg]  153.8  228.18 297.7  1097 CTDOXY      

* Data-quality Flags

    pressure:    "2" 1097
    temperature: "2" 1096, "9" 1
    salinity:    "2" 1096, "9" 1
    oxygen:      "2" 1093, "9" 4

* Processing Log
    - 2017-02-17 19:19:57 UTC: `create 'ctd' object`
    - 2017-02-17 19:19:57 UTC: `read.oce(file = "18HU20130507_00235_00001_ct1.csv")`
    - 2017-02-17 19:21:51 UTC: `handleFlags(flags=c(1, 3, 4, 5, 6, 7, 8, 9), actions=NA)`
> 
```

**Comment by dankelley** [comment]
Another oddness I'll address: in this and the test file for #1194 there are two different units stated for temperature (see below) so I'll want to match both to a common unit (unless there really are two variants)
```
$ grep "68,,"  ~/git/oce/tests/testthat/local_data/18*
/Users/kelley/git/oce/tests/testthat/local_data/18HU2010014_00003_00001_ct1.csv:DBAR,,IPTS-68,,PSS-78,,UMOL/KG,
/Users/kelley/git/oce/tests/testthat/local_data/18HU20130507_00235_00001_ct1.csv:DBAR,,ITS-68,,PSS-78,,UMOL/KG,
```

**Comment by dankelley** [comment]
Re the name of the unit, I think the correct is `IPTS-68`, at least according to my searching e.g. below are two authorative-looking treatments that mention IPTS-68. (I didn't read these in the last minute ... I'll return later, maybe, to skim them.)
* http://digicoll.manoa.hawaii.edu/techreports/PDF/NBS125.pdf
* http://nvlpubs.nist.gov/nistpubs/jres/77A/jresv77An3p309_A1b.pdf

**Comment by richardsc** [comment]
I'm 99% sure the correct unit is IPTS-68. The latter looks like a typo, where it was confused with the more modern ITS-90.

**Comment by dankelley** [comment]
Agreed on IPTS-68. I will push a solution to this issue in an hour or so, when local tests are done. I am doing something quite tricky in detecting S and T missing values (look for minima, see if they are under -8, see if either they agree or if only S or T has such a value...) and if it sets a missingValue then it will spit a warning and insert a comment in the processingLog.

**Comment by dankelley** [comment]
Fixed in `develop` commit 2b6fd049dcc16e8d1f199ca2e361bc1464f710fd. The new behaviour for `missingValue` is documented, so I won't explain it further here. I ran my local tests, without seeing problems, and I ran both this test and 1158 and both are ok.
